Sources/CardPayments/CardError.swift

```diff
@@ -39,7 +39,7 @@ public enum CardError {
         case malformedDeeplinkURLError
 
         /// 10. Cancellation from 3DS verification
-        case threeDSCancellationError
+        case threeDSecureCanceledError
```

Comment:
This is a random side nit: I noticed the `Error` suffix may be a bit redundant here since the type is `CardError`. I wonder if it's worth cleaning up in a future release.

Reply:
Oh yeah that definitely will change on Android. They shouldn't have been named that way.